src/go/k8s/controllers/redpanda/cluster_controller_test.go

```diff
@@ -113,8 +124,10 @@ var _ = Describe("RedPandaCluster controller", func() {
 
 			Expect(sts.Spec.Template.Spec.Containers[0].Resources.Requests).Should(Equal(resources))
 			Expect(sts.Spec.Template.Spec.Containers[0].Resources.Limits).Should(Equal(resources))
-		})
-	})
+		},
+		Entry("Single pod", int32(1)),
+		Entry("Two pods", int32(2)),
```

Comment:
If I am not missing something - I am actually not sure this is even testing something to be honest. In integration env there are no controllers running so this test is only making sure we pass this number into statefulset spec (there won't be any pods created), if we need to test both 1 and 2 for that, maybe it would be better suited on the unit level. And then maybe e2e level

Reply:
no controllers from core controllers, to be precise

Reply:
Hmm. I'll drop this commit.